R/informalr-package.R

```diff
@@ -0,0 +1,10 @@
+#' @keywords internal
+"_PACKAGE"
+
+# The following block is used by usethis to automatically manage
+# roxygen namespace tags. Modify with care!
+## usethis namespace: start
+#' @importFrom magrittr %>%
+#' @importFrom stats na.omit
```

Comment:
I think importing the pipe is necessary, as one cannot call it from its namespace in a function. Importing `stats::na.omit`, however, feels unnecessary: one can simply use `stats::na.omit`. Allowing this would create a slippery slope, in which in the end all functions we use are imported. Or does the `@importFrom stats na.omit` have another side effect I am unaware of?

Reply:
Good point. Let's avoid slippery slopes. In the official R package doc, they also discourage the use of @importFrom.